controllers/bootstrap/init.go

```diff
@@ -420,6 +421,33 @@ func (b *Bootstrap) CreateNsScopeConfigmap() error {
 	return nil
 }
 
+// CreateNsScopeConfigmap creates nss configmap for operators
```

Comment:
comment does not match function name nor purpose

Reply:
Nice catch!

Reply:
It is updated